README.md

```diff
@@ -34,24 +168,17 @@ scrape_configs:
 
   # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
   - job_name: 'maia'
-    metrics_path: "/api/v1/metrics"
+    metrics_path: "/federate"
```

Comment:
/federate is strictly not part of the API. The idea is to exactly provide the same API as Prometheus with the only exception being the authentication. That way I can use the same CLI commands also for querying Prometheus.